recipes/fftw/all/conanfile.py

```diff
@@ -98,11 +98,18 @@ def package_info(self):
         cmake_target_name = "fftw3" + prec_suffix
         pkgconfig_name = "fftw3" + prec_suffix
         lib_name = "fftw3" + prec_suffix
+        
+        self.cpp_info.set_property("cmake_file_name", cmake_config_name)
+        self.cpp_info.set_property("cmake_target_name", "{}::{}".format(cmake_namespace, cmake_target_name))
```

Comment:
```suggestion
```
I think we can remove this and leave the component one but adding a #FIXME explaining that in Conan 2.0 we should undo the workaround and move that component to be the root cpp_info.